doc/parameters/param_list.md

```diff
@@ -175,7 +175,18 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | `<filter name>`.enabled | true | Whether it is enabled |
-| `<filter name>`.filter_info_topic | N/A | Name of the CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.filter_info_topic | N/A | Name of the incoming CostmapFilterInfo topic having filter-related information |
```

Comment:
Needs the corresponding nav2 docs PR to add these parameters, the plugin to the plugins page, migration guide, etc.

Reply:
For costmap2d and also controller server's topic.

Reply:
Created https://github.com/ros-planning/navigation.ros.org/pull/122 for this